### ISSUE TITLE
Editor's note treatment is coded into body copy

### ISSUE THREAD

**Issue by alexgreen** [TheTyee/design-article.thetyee.ca#224]
In ye olden days editors had the unsightly habit of writing "[Editor's note:" at the start of an ed's note and ending the note with "]". As here: http://preview.thetyee.ca/News/2016/01/01/Coast-Today-Toast-Tomorrow/

It has been decided (in consultation with editorial I believe) that this practice will be abolished and that staff will undergo pavlovian retraining.
But did we decide what to do about the offending text in all the legacy articles? Is there some way to get rid of all those hand-written Ed note labels?

**Comment by phillipadsmith** [comment]
> But did we decide what to do about the offending text in all the legacy articles? 

I think @MrBryan wrote some magic to catch these...?

**Comment by MrBryan** [comment]
I wrote some magic to catch them and apply the css treatment but not strip the "[Editor's note]" but it would be easy enough to do that if that's what we want to do

**Comment by alexgreen** [comment]
Yes, please cast your magic spell to banish the "[Editor's note]" from our fair kingdom!

**Comment by alexgreen** [comment]
@MrBryan , it looks like some of the editors are immune to your magic. For example this Ed's note is still written into the body copy and not receiving proper formatting:

http://thetyee.ca/Culture/2016/07/15/New-Glimpses-of-Fighting-Fascists/
